src/fragments/lib/graphqlapi/ios/subscribe-data.mdx

```diff
@@ -103,12 +118,10 @@ Calling `cancel()` on the subscription will disconnect the subscription from the
 
```

Comment:
i think we need to verify this statement
> Calling `cancel()` on the subscription will disconnect the subscription from the backend. Any downstream subscribers will also be cancelled. On the other hand, calling `cancel()` on the Combine subscriber (e.g., the `AnyCancellable` returned from `sink()` will cause that Combine subscriber to stop receiving updates, but any other attached subscribers will continue to receive subscription data. For example:

I wonder if this is still true, if the combine subscriber gets cancelled, i think it will cancel the sequence and other attached subscribers won't continue to receive subscriptin data anymore

feel free to create a new issue for this to verify this section of the doc separately